index.yaml

```diff
@@ -132,8 +132,19 @@ sources:
     homepage: https://raw-data.gitlab.io/post/malsilo_2.1/
     checksum: true
 
+  stamus/lateral:
```

Comment:
As this seems specific to Windows, should that be reflected in the name? If the plan is maybe to be less specific than Windows at some point in the future, best not to embed Windows in the naming.